charts/ai-service/templates/artifactsCache.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.artifactsCache.enabled -}}
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{ include "common.fullname" . }}-artifacts
+  labels:
+    {{- include "common.labels" . | nindent 4 }}
+spec:
+  accessModes:
+    - ReadWriteMany
```

Comment:
this should be settable as well (use RWX as default)
why? 
not all storage classes support all accessModes